english/make-friends/where_am_i/where_am_i.md

```diff
@@ -0,0 +1,25 @@
+You realise you need as much information as you need to get out of this situation.
+
+And the only thing you can ask about this seems to be Marshmellow man. You decide to befriend him.
+
+"Hello, Marshmellow man..er.. uh..Do you have a name?" , I asked, not really expecting any answer 
+and wondering whether I have gone crazy to be talking with a marshmellow.
+
+"My name is Marshy", mellowed a sweet sounding sing songy voice. "And I am a woman", it said.
+
+I was a little embarrassed and surprised to hear back a reply. "Where am I?", I asked.
+
+"Whoa..an alien planet? How did I end up here?", I asked with my heart rate begining to rise.
+
+"Don't  be afraid", Marshy tried to calm me down. "This place is friendly. This planet is called UR58F3 and about 10 light years
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.